### ISSUE TITLE
mubu.process with externally generated markers

### ISSUE THREAD

**Issue by einbond** [ircam-ismm/catart-mubu#82]

**Comment by jln-** [comment]
Hi. Assuming I understand correctly, this can be done using chop segmenter's segtimes and segdurations attributes. FWIW, below is an little patch to a list of markers from a csv file exported from Reaper.

<pre><code>
----------begin_max5_patcher----------
2251.3ocybszrahiEds8uBFWyht6xsKzCv3rp6pmEy9opYS5TtjAYeIAiX.w
MNIU9uO5AfAa.KigquNULXIgz47oyKIct7i4yVrichlsv5CVezZ1reLe1LUQ
xBlU76YKNRN4GQxTMaQL8qrcedwRcUb5Itp3DqCojcaORR+BMMa69T1ws9Yu
V1tnvXpOKOV0XXQgw4GY47HJW0yfhRSHb+WBiOrMk5y0Tli8J6kV.3F0Er5W
HmU1Vep3YzcC+aIT8CrXGI9vhppCCTznft+80KNO3gwsN1zzB1uf+msXeXD8
UAaExhq05YKHII0JdVsGQBZelo5HukUEEFqKxopnT5qgkOOtpTRp.A3B1OOU
wPKN4hWbtaXAzz37PEonK7myKII0DUL4HMKg3qeX47YY00vTEX5Aje6pgTOz
lyXpPN3PDy+KTE1YWVHKgFGFmjRynwbBufzqpNftmjGw2tmEyyB+th..vUsV
+9BBr0Jkrfh5+yzPRTE8eHMLfEKIhFSDxhKGtOZAbTxIN0YFUKhIIs7vBwBA
pzQkYBlLOaGIUNOsKhVS9UH8yXQMqp54hn64EUmDFGeAJxYIcWYZ3gW54Y2w
DUdru9VUS117XcsaEhD7sYjWah1bRTTgxbyt+DIN7HgS4g5o.ncUkzXhfQeI
yOkEE0fe007ZK0DHDw8oeMLf+hZfpKLHZdXRoPzhpY4fvCzLdyx3jCYMKIi+
MMnWqn7cEpva4ziIQBtnYCDJGgY7rWXeMqngkBZ0AfyVDqqRW2xXix6yB4EV
ISoBAr.KggCwk5MoUagcYOD.vq7Pm+f8bVZggJc4MtxugvZB+sZgbQipqYgD
eAYUyJYY4+b97xaVN9fzw7c4q3oD+uX4yRSxy1RCB4VEtV5FzPC.zbr09T7r
MD0VV7+tPOP2nWsJjJiAa0Fd1R37zvc4bsT2rq.tYKDNY1wxnMlCNOILwSGb
qPqvtQcX+nNBpQVjmBmssW4XBNGJhTXo9RmHMB7rkT+dzJpeXTlEXv3iNzFH
Po653bGRgchK.Tm3B7sRj4+kSS+lUjkeDkjNXU1h39PJvACMzxlZrKUT0DP2
Xk8ahLjHbrLxAZqfkLzXq+IXvNCPNPU7v.jBkvOn4e3yV3I+6gB6DVvAK0TZ
xwSIu3.LCPTqXX44q8Y5wc8yFj989vGfY3i1liXQUFaQtS7v4YiGkAVkRICO
rpByM15ECY+nwQY+rcOot4As8tFp0k1XtmISzffduC7cmkebv9saMZRjqqTY
BiFA23nmtg3OyBisPi6hTPHc71PuU0pBAc7dHcsdBED8F40J9VvE91vkSkWK
fGxLyxUKHoWYomtonT5A5oDqe4uEeB9se8Ck2rp7ltwMm9wMMdAcA5PEQ245
3tEzAgO8.Dc+fiPXvYyv0Ds8tVS7dWxQm.j8y1N0uIcYauZ3vi6lqMFUFN8Z
yfm8QLROKUE+7wHW6GCjzlmJfEGzX.KduInhpWevXBKNxgpuGntR2gIaWqBe
1wiz3la+opihCnmFaCOZhua3w9Fqk.oWj0Fmd.n5qTvnsGaPHv485Wth5N1A
WEKHqucnIikm5WNMVHhZ0jYBnY7v3pS93imC.Q1PilYtWh.ZHQ.8lPhPtlWK
nITwTRDNFBEfIcBQtzNyHCf8TRFlRESHMfLU3DOkDAvT0zoT3DYpTg2TSDPS
sXAmRp.Xhn4TJVnrGZBVTZVY5nByLVflRx.Z3TBZJMYAMUGANkZp.783VGLU
TgwlsT9dmLvPI1Ad5yI.62Cg4nICv6Cx.Zty8Zs6XXPBKLlWDBLtXSI23Iu.
f3FK1Um5.x5lHNwzH1v1ScvqlLo51IQTTXYlVcqiv+GCIGWNDw1QhtHwTZKE
Xlel3TeqwnGH88Bn6ooQrutXfol2Z0oH4b0gO1yhiarAksmUdCf+hBektJHk
kT1iuRNijWUY87XqFSo3IMi43sZCvU7Yi9iKPtM+qqwgATeV.UlTd0gpVOLg
d1fATsTaz0tW.bYGfHDTwVjTAKyooa0okUcBKhdPl5QxG5eIPhhzpwRlhiV+
xp+5+7e+Uq+MMk9O5OOIuH++.MTJpzF5Qu3URTNks+57l7LsGF2Rh0UuAQr3
CWO29Q3mVzRiydgkx6TRnQSKAa775FAFS0sTwrJ0JfESsHQQxqsq3g5VtAgT
a4hKTcPAxs381pdK6Q5AztNHZ7X5xyn0+EVxpL5gf7Tk.T1cazAoObVzFnVG
cnlcvaFM6NcvyFkKaljcznaiF5T+.fVe6ziteIA.rWkeC87cURqMlJPQLRf9
bluWQmMJEErvDtzgkAZMclJ4PGmQS7o44+blOU5Jh91RPl56ImptOiGDPe0x
tUwHzPQlM.Ix3oO9iM2sNEzA2Jp.e2DrBBa+PAqHr0N0lM3V6j+6tcJTjMgq
06FOdigR2Ku3ZqgnsdpY5K8OHy+7622.Tm8+k9Ft9.gM02v5IW49ffJj4X+P
YQ0eXEky4CfCQSsh5ag2uRvXr79gAueb9I5ZtH.6AZlas9rqwkeO.WbfdWRg
8aQvSIoLeg9SCAH0hp2tOJ+jxXg0eHiq2pbo1rzLq+n7wrk2RSHAx6TVVJC6
zBTTfz8Z08B2qU2SImqnvWq3mRUVwJwOPCpR66lhv3Vmuv2bIftEmgJVGWuI
RvEoIe6FrcFM06tLfcjHj+O4yhjgjHrlo+8fCmWaOC.GrIa2Qiiabr3sej38
snr0cbb3M9KQD2qglVNC7KN+6BVR02MOyaMGd4ljUvmWum7M1BwN1XrZ6.8U
PooCzZCFGzHLNHSFmwfg.lLRqwivHIWo5s4oF605k6CLxSsGcXriJ52y+3QI
s0OLoAg3RpA1f1fOLw4Z.woVv6iOEYhv.FLFijIJr3wP.G0HoO5ZjbGqQBbq
QBMVizMEIr8FigRJ0it0PMFLkZuHtIWoVa9HLV36Pi+wMvBdKLvVYCneAvMi
xP8lAe1dFY96pgRGSwEudCjixEuVCt3UZv0uNC59UYvkuFCTGrVWu9Bl+y4+
etHTO6A
-----------end_max5_patcher-----------
</code></pre>

**Comment by christophertrapani** [comment]
Yes, thank you, Julien! I think that's (at least part of) the solution I
was looking for...

**Comment by diemoschwarz** [comment]
Hi all, coming back to this, if you think it would be usefull, I could see 3 steps to generalise this:
- [ ] add a mubu module and tutorial to import various marker file formats (reaper, audacity, etc.) into a mubu track
- [ ] add a catart module to use a marker track for segmentation
- [ ] add and document real-time external segmentation (as `camu.pipo.descr-external`?)
